python-sdk/src/langwatch/evaluations.py

```
@@ -142,38 +153,47 @@ def prepare_data(
+        dataDict["contexts"] = contexts
     if expected_contexts is not None:
-        data["expected_contexts"] = expected_contexts
+        warn("For the `evaluate` or `async_evaluate` function, the `expected_contexts` argument is deprecated and will be removed in a future version. Future versions of the SDK will not support it. Please use the `data` argument instead, to have complete control over the data structure.")
+        dataDict["expected_contexts"] = expected_contexts
     if conversation is not None:
-        data["conversation"] = conversation
+        warn("For the `evaluate` or `async_evaluate` function, the `conversation` argument is deprecated and will be removed in a future version. Future versions of the SDK will not support it. Please use the `data` argument instead, to have complete control over the data structure.")
+        dataDict["conversation"] = conversation
+
     if trace_id is not None:
         warn(
```

Ruff (0.8.2)

No explicit `stacklevel` keyword argument found

(B028)